BLSQ/iaso
# Wc2-90: beneficiaries export to xls, date are in timestamp

When exporting beneficiaries list in Excel, dates (created at/updated at) were displayed as timestamps rather than a human-readable date time

## Self proof reading checklist


## Changes

Added basic datetime formatting

## How to test

Create an XLSX export of beneficiaries, have a look at the content and make sure thedates are readable.
